src/helpers/taxonomy-helper.php

```diff
@@ -36,4 +36,15 @@ public function __construct( Options_Helper $options_helper ) {
 	public function is_indexable( $taxonomy ) {
 		return ! $this->options_helper->get( 'noindex-tax-' . $taxonomy, false );
 	}
+
+	/**
+	 * Retrieves the term description (without tags).
+	 *
+	 * @param int $term_id Term ID.
+	 *
+	 * @return string Term description (without tags).
+	 */
+	public function get_term_description( $term_id ) {
+		return \wp_strip_all_tags( \term_description( $term_id ) );
```

Comment:
Have you considered moving the `wp_strip_all_tags` to a string helper class?

Reply:
Just like `$this->string->strip_all_tags`. This makes the function more reusable like for the post excerpt as well.

Reply:
Right, this is the second application like this. Good idea.